config.js

```diff
@@ -1,4 +1,10 @@
 module.exports = {
+  "flags": {
+    "injectTestUrls": true,
```

Comment:
We have to remember to turn this back to false before having the final version.
Or change the default urls in case we want to add a default one explaining the app.